Src/FluentAssertions/Numeric/NumericAssertions.cs

```diff
@@ -474,7 +474,7 @@ public AndConstraint<TAssertions> NotBeOfType(Type unexpectedType, string becaus
 
     /// <inheritdoc/>
     public override bool Equals(object obj) =>
-        throw new NotSupportedException("Calling Equals on Assertion classes is not supported.");
+        throw new NotSupportedException("Equals is not part of Fluent Assertions. Did you mean Be() instead?");
```

Comment:
I think for this and boolean, we just need to mention `Be()`, right?

Reply:
Agree